.devcontainer/devcontainer.json

```diff
@@ -1,14 +1,14 @@
 {
-  "build": {
-    "dockerfile": "./Dockerfile",
-    "context": "."
-  },
+  "dockerComposeFile": "./docker-compose.yml",
+  "service": "auto-gpt",
+  "workspaceFolder": "/workspace/Auto-GPT",
+  "shutdownAction": "stopCompose",
   "features": {
     "ghcr.io/devcontainers/features/common-utils:2": {
       "installZsh": "true",
       "username": "vscode",
-      "userUid": "1000",
-      "userGid": "1000",
+      "userUid": "6942",
+      "userGid": "6942",
```

Comment:
does this give permission issues?

Reply:
This is to avoid bumping up with the user created by the dockerfile in the project root, otherwise there are ownership issues as the appuser has a UID/GID of 1000

Reply:
It's to keep the GID/UID of the devcontainer's default vscode user from bumping up against the appuser profile created in the other dockerfile